cmd/admin-handlers.go

```diff
@@ -1217,6 +1217,7 @@ func (a adminAPIHandlers) ObjectSpeedTestHandler(w http.ResponseWriter, r *http.
 	storageClass := strings.TrimSpace(r.Form.Get(peerRESTStorageClass))
 	customBucket := strings.TrimSpace(r.Form.Get(peerRESTBucket))
 	autotune := r.Form.Get("autotune") == "true"
+	keepData := r.Form.Get("keep-data") == "true"
```

Comment:
```suggestion
	noClear := r.Form.Get("noclear") == "true"
```

Just like warp @vadmeste